README.md

```diff
@@ -113,7 +112,7 @@ an endpoint for subscription, you can use expanded version:
 }
 ```
 
-> Note: do not save secure information in .graphqlconfig file. Use [Environment variables](specification.md#referencing-environment-variables) for that like in the example above.
+> Note: do not save secure information in your .graphql file. Use [Environment variables](specification.md#referencing-environment-variables) for that like in the example above.
```

Comment:
Is this usage correct?

Reply:
Good catch! I need to work on my find/replace skills 😄